geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -134,12 +133,12 @@ data.
 ## <a id="loss-of-connections" class="no-quick-link"></a>Loss of Connections
 
 There are a number of events that might occur within the <%=vars.product_name%> cluster that can result
-in the cluster closing the connection to the Redis client. Losing the connection to the cluster does not
+in the cluster closing the connection to the Compatible with Redis Client. Losing the connection to the cluster does not
```

Comment:
Same as above, I think this could be `Redis client`.  Same goes for the two instances below this one.